Dockerfile

```
@@ -0,0 +1,21 @@
+FROM cvriend/pgs:latest
+WORKDIR /
+
+# Need fslinstaller.py
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y wget bc dc tree parallel zip && rm -rf /var/lib/apt/lists/*
```

```suggestion
RUN apt-get update && \
    apt-get install -y --no-install-recommends \
    wget \
    bc \
    dc \
    tree \
    parallel \
    zip && \
    rm -rf /var/lib/apt/lists/*
```
Could add the `--no-install-recommends` flag to `apt-get-install` so optional packages are not installed (which might reduce image size a little bit)

---

Sounds good - I'll add this!